src/components/BottomBar/SkyBrowser/SkyBrowserNearestImagesList.jsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import { useSelector, shallowEqual } from 'react-redux';
+import PropTypes from 'prop-types';
+import CenteredLabel from '../../common/CenteredLabel/CenteredLabel';
+import {FilterList, FilterListData} from '../../common/FilterList/FilterList';
+import styles from './SkyBrowserNearestImagesList.scss';
+import SkyBrowserFocusEntry from './SkyBrowserFocusEntry';
+
+function SkyBrowserNearestImagesList({
+  activeImage,
+  height,
+  selectImage,
+  currentBrowserColor
+}) {
+  const fov = useSelector((state) => {
+    return state.skybrowser.browsers[state.skybrowser.selectedBrowserId].fov;
+  }, shallowEqual);
+  const ra = useSelector((state) => {
+    return state.skybrowser.browsers[state.skybrowser.selectedBrowserId].ra;
+  }, shallowEqual);
+  const dec = useSelector((state) => {
+    return state.skybrowser.browsers[state.skybrowser.selectedBrowserId].dec;
+  }, shallowEqual);
+  const cartesianDirection = useSelector((state) => {
+    return state.skybrowser.browsers[state.skybrowser.selectedBrowserId].cartesianDirection;
+  }, shallowEqual);
+  const luaApi = useSelector((state) => {
+    return state.luaApi
+  }, shallowEqual);
+  const imageList = useSelector((state) => {
+    return state.skybrowser.imageList
+  }, shallowEqual);
+
+  const distanceSortThreshold = 0.1;
+
+  function euclidianDistance(a, b) {
+    let sum = 0;
+    for (let i = 0; i < 3; i++) {
+      sum += distPow2(a[i], b[i]);
+    }
+    return Math.sqrt(sum);
+  }
+
+  function distPow2(a, b) {
```

Comment:
And here they are distances? Or vectors? Or what is a and b? Please use more meaningful names. :)